lib/common/common.go

```diff
@@ -51,14 +51,10 @@ func ParseDockerURL(arg string) *types.ParsedDockerURL {
 	}
 	indexURL, imageName := SplitReposName(taglessRemote)
 
-	if indexURL == "" && !strings.Contains(imageName, "/") {
+	if indexURL == defaultIndexURL && !strings.Contains(imageName, "/") {
```

Comment:
Alright. BTW, here's the code in Docker: https://github.com/docker/docker/blob/117a982d2e805d3279ff75fcec071635e5191ef6/reference/reference.go#L171